FortnitePorting/Plugins/Blender/import_task.py

```diff
@@ -563,14 +591,153 @@ def get_meta(search_props):
+
+                        if not shape_keys:
+                            # Create Basis shape key
+                            imported_mesh.shape_key_add(name="Basis", from_mix=False)
+
+                        # NOTE: I think faceAttach affects the expected location
+                        # I'm making this assumption from observation of an old
+                        # export of face poses for Polar Patroller.
+                        loc_scale = (0.01 if self.options.get("ScaleDown") else 1)
+                        for pose in pose_data:
+                            influences = pose['Keys']
+
+                            # If there are no influences, don't bother
+                            if not influences:
+                                continue
+
+                            # Enter pose mode
+                            bpy.context.view_layer.objects.active = armature
+                            bpy.ops.object.mode_set(mode='POSE')
+
+                            # Reset all transforms to default
+                            bpy.ops.pose.select_all(action='SELECT')
+                            bpy.ops.pose.transforms_clear()
+                            bpy.ops.pose.select_all(action='DESELECT')
+
+                            # Move bones accordingly
+                            for bone in influences:
+                                bone_name = bone['Name']
+
+                                pose_bone: bpy.types.PoseBone = armature.pose.bones.get(bone_name)
+                                if not pose_bone:
+                                    # For some reason, some of the bones start capitalized and some dont
+                                    bone_name = bone_name[0].capitalize() + bone_name[1:]
+                                    pose_bone = armature.pose.bones.get(bone_name)
+                                if not pose_bone:
+                                    # For cases where pose data tries to move a non-existant bone
+                                    # i.e. Poseidon has no 'Tongue' but its in the pose asset
+                                    if not is_face:
+                                        # There are likely many missing bones in FaceAcc, but we
+                                        # process as many as we can.
+                                        Log.warn(f"could not find: {bone_name} for pose {pose['Name']}")
+                                    continue
+
+                                rotation = bone['Rotation']
+                                assert rotation['IsNormalized'], "non-normalized rotation unsupported"
+
+                                edit_bone = pose_bone.bone
+                                post_quat = Quaternion(post_quat) if (post_quat := edit_bone.get("post_quat")) else Quaternion()
+
+                                assert pose_bone.rotation_quaternion == Quaternion(), "initial pose bone is not identity"
```

Comment:
I think I'll just call `pose_bone.matrix_basis.identity()` and get rid of these assertions

Reply:
bet.